docs/api-reference/supervisor-api.md

```diff
@@ -23,89 +23,3075 @@ sidebar_label: Supervisors
   ~ under the License.
   -->
 
-This document describes the API endpoints to manage and monitor supervisors for Apache Druid.
+This document describes the API endpoints to manage and monitor Supervisors for Apache Druid.
 
-## Supervisors
+In this document, `http://<SERVICE_IP>:<SERVICE_PORT>` is a placeholder for the server address of deployment and the service port. For example, on the quickstart configuration, replace `http://<ROUTER_IP>:<ROUTER_PORT>` with `http://localhost:8888`.
 
-`GET /druid/indexer/v1/supervisor`
+## Supervisor information
 
-Returns a list of strings of the currently active supervisor ids.
+### Get an array of active Supervisors
 
-`GET /druid/indexer/v1/supervisor?full`
+#### URL
 
-Returns a list of objects of the currently active supervisors.
+<code className="getAPI">GET</code> `/druid/indexer/v1/supervisor`
+
+Retrieves an array of active Supervisors. Returns a response object that includes an array of strings representing the names of the active Supervisors. If there are no active Supervisors, it returns an empty array.
+
+#### Responses
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--200 SUCCESS-->
+<br/>
+*Successfully retrieved array of active Supervisor IDs* 
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+---
+
+#### Sample request
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--cURL-->
+```shell
+curl "http://<ROUTER_IP>:<ROUTER_PORT>/druid/indexer/v1/supervisor"
+```
+<!--HTTP-->
+```HTTP
+GET /druid/indexer/v1/supervisor HTTP/1.1
+Host: http://<ROUTER_IP>:<ROUTER_PORT>
+```
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+#### Sample response
+
+<details>
+  <summary>Click to show sample response</summary>
+  
+  ```json
+  [
+    "wikipedia_stream",
+    "social_media"
+  ]
+  ```
+</details>
+
+### Get an array of active Supervisor objects
+
+#### URL
+
+<code className="getAPI">GET</code> `/druid/indexer/v1/supervisor?full`
+
+Retrieves an array of active Supervisor objects. Each object has properties relevant to the Supervisor: `id`, `state`, `detailedState`, `healthy`, and `spec`. 
```

Comment:
```suggestion
Retrieves an array of active supervisor objects. Each object has properties relevant to the Supervisor: `id`, `state`, `detailedState`, `healthy`, and `spec`. 
```
```suggestion
Retrieves an array of active supervisor objects.
```
I don't think you need to list the properties in this paragraph since you provide detailed information about the properties in the table that follows.
What happens if there are no active supervisors? Perhaps it is best to remove the word "active" completely? Same comment applies to `GET` `/druid/indexer/v1/supervisor`.